framework/wazuh/core/cluster/worker.py

```diff
@@ -803,6 +812,34 @@ def overwrite_or_create_files(filename_: str, data_: Dict):
                 result_logs["debug2"][directory].append(f"Error removing directory '{directory}': {e}")
                 continue
 
+        # If analysisd must be reloaded
+        if len(reload_ruleset_files) > 0:
+            try:
+                response = analysis.send_reload_ruleset_msg(origin={'module': 'cluster'})
+                if response.is_ok():
+                    if response.has_warnings():
+                        for warning in response.warnings:
+                            matched = [file_info for file_info in reload_ruleset_files if file_info['file'] in warning]
+                            if matched:
+                                for match in matched:
+                                    result_logs['debug2'][match['file']].append(warning)
+                            else:
+                                result_logs['debug2']['warnings'].append(warning)
+
+                else:
+                    for error in response.errors:
+                        matched = [file_info for file_info in reload_ruleset_files if file_info['file'] in error]
+                        if matched:
+                            for match in matched:
+                                result_logs['error'][match['type']].append(f"Error reloading {match['type']} "
+                                                                      f"file '{match['file']}': {error}")
+                        else:
+                            result_logs['generic_errors'].append(f"Error reloading ruleset {error}")
+
+            except Exception as e:
+                result_logs['generic_errors'].append(f"Error reloading ruleset {e}")
```

Comment:
We are missing modifying the `errors` dictionary to print the "Found errors: ..." log